jobserver/views/projects.py

```diff
@@ -99,6 +101,35 @@ def post(self, request, *args, **kwargs):
         return redirect(project)
 
 
+@method_decorator(require_superuser, name="dispatch")
+class ProjectRemoveMember(View):
+    model = ProjectMembership
+
+    def post(self, request, *args, **kwargs):
+        try:
+            membership = ProjectMembership.objects.select_related("project").get(
+                project__org__slug=self.kwargs["org_slug"],
+                project__slug=self.kwargs["project_slug"],
+                user__username=self.request.POST.get("username"),
+            )
+        except ProjectMembership.DoesNotExist:
+            raise Http404
+
+        can_manage_members = has_permission(
+            self.request.user,
+            "manage_project_members",
+            project=membership.project,
+        )
+        if can_manage_members:
+            membership.delete()
+        else:
+            messages.error(
+                request, "You do not have permission to remove Project members."
```

Comment:
I don't think "Project" should be capitalised here.  I understand why you have, but I think model names should only be capitalised in code (obvs) and developer docs (maybe), but not in anything user-facing.